packages/firestore/src/remote/connection.ts

```diff
@@ -38,14 +38,15 @@ export interface Connection {
    * representing the JSON to send.
    *
    * @param rpcName - the name of the RPC to invoke
-   * @param path - the path to invoke this RPC on
+   * @param path - the path to invoke this RPC on. An array of path segments
+   * that will be encoded and joined with path separators when required.
    * @param request - the Raw JSON object encoding of the request message
    * @param token - the Token to use for the RPC.
    * @returns a Promise containing the JSON object encoding of the response
    */
   invokeRPC<Req, Resp>(
     rpcName: string,
-    path: string,
+    path: string[],
```

Comment:
Maybe bikeshedding a bit here, but I just looked closer at ResourcePath and it basically has no dependencies. It would definitely be clearer to just take a ResourcePath as the argument here, rather than a string array. What do you think?